### PR TITLE
incorprate reticulate traceback into rstudio error handler traceback

### PR DESCRIPTION
**Pull request by jjallaire**

### CODE REVIEW

**Comment by jjallaire** [comment]
@terrytangyuan Once this hits the daily build (should be tomorrow) the Python tracebacks will be included in the RStudio traceback UI (including being able to click and navigate to the lines of code on the stack)

**Comment by terrytangyuan** [comment]
This is life saver. Thanks JJ!

File: src/cpp/session/modules/SessionErrors.R [hunk 1]
```diff
@@ -77,16 +77,67 @@
...
+   grepl("^py_call_impl\\(", func) && 
+         .rs.isPackageVersionInstalled("reticulate", "0.7.0.9005") &&
+         !is.null(reticulate::py_last_error())
+})
+
+.rs.addFunction("getActivePythonStackTrace", function() {
+   stack <- list()
+   # silence errors so anything unexpected doesn't mess with global RStudio error handling
+   tryCatch({
+      traceback <- reticulate::py_last_error()$traceback
```
Comment (terrytangyuan) [inline src/cpp/session/modules/SessionErrors.R]: I now see the true reason for introducing this `reticulate::py_last_error()` @jjallaire

**Comment by jjallaire** [comment]
Yes, `reticulate::py_last_error()` was part 1 of 2 commits and it just took me longer than I thought it would to get part 2 in! :-)

**Pull request merged**